README.md

```diff
@@ -0,0 +1,42 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Book list, Add book, Remove book, navigation bar, OnePage tech
+
+![screenshot](./screenshot.png)
+
+## Live Demo
```

Comment:
- [ ] [OPTIONAL] Please, try to fix the demo live link - actually it is broken 👍